### PR TITLE
new changes from collab1

### PR DESCRIPTION
**Pull request by kumar88sunil**

### CODE REVIEW

File: file1 [hunk 1]
```diff
@@ -1 +1 @@
-hello world
```
Comment (vistasunil) [inline file1:1]: why this line delete?

File: file7 [hunk 2]
```diff
@@ -0,0 +1,4 @@
+dkjjndakksjdkallksnjksalakja
```
Comment (vistasunil) [inline file7:1]: move this later.

**Review by vistasunil — CHANGES_REQUESTED** [review]
rework needed

File: file1 (hunk 1 above)
Comment (kumar88sunil) [reply file1:1]: this was as per plan

File: file7 (hunk 2 above)
Comment (kumar88sunil) [reply file7:1]: ok

**Review by vistasunil — APPROVED** [review]

**Pull request merged**